Update MPSL and SoftDevice Controller (#1068)
nrfconnect/sdk-nrfxlib: mpsl/CHANGELOG.rst

=======

* FEM-related firmware is now built and deployed into separate libraries, one for each FEM implementation (such as ``simple_gpio``).

> * Users of mpsl have to call :c:func:`mpsl_calibration_timer_handle` when calibration shall occur.

Reviewer: ```suggestion
* The :c:func:`mpsl_calibration_timer_handle` function must now be called when calibrating.
```
